### ISSUE TITLE
context menu in AgGrid with popupParent 

### ISSUE THREAD

**Issue by giodegas** [elimintz/justpy#131]
It would be nice to use the _Popup Parent_ way to make right-click pop-up context menù from AgGrid cells as documented here:

https://www.ag-grid.com/javascript-grid-context-menu/#popup-parent

I tried just adding the grid option to this examples, but cannot make it to work:

```python
import justpy as jp
import pandas as pd

wm_df = pd.read_csv('https://elimintz.github.io/women_majors.csv').round(2)

def row_selected(self, msg):
    print(msg)
    if msg.selected:
        self.row_data_div.text = msg.data
        self.row_selected = msg.rowIndex
    elif self.row_selected == msg.rowIndex:
        self.row_data_div.text = ''

def grid_test():
    wp = jp.WebPage()
    row_data_div = jp.Div(a=wp)
    grid = wm_df.jp.ag_grid(a=wp)
    grid.row_data_div = row_data_div
    grid.on('rowSelected', row_selected)
    grid.options.columnDefs[0].checkboxSelection = True
    grid.options.popupParent = "document.querySelector('body')"
    return wp

jp.justpy(grid_test)
```

neither this way:

```python
import justpy as jp

grid_options = """
{
    defaultColDef: {
        filter: true,
        sortable: true,
        resizable: true,
        cellStyle: {textAlign: 'center'},
        headerClass: 'font-bold'
    },
      columnDefs: [
      {headerName: "Make", field: "make"},
      {headerName: "Model", field: "model"},
      {headerName: "Price", field: "price"}
    ],
      rowData: [
      {make: "Toyota", model: "Celica", price: 35000},
      {make: "Ford", model: "Mondeo", price: 32000},
      {make: "Porsche", model: "Boxter", price: 72000}
    ],
    popupParent: "document.querySelector('body')"
}
"""

def grid_test():
    wp = jp.WebPage()
    grid = jp.AgGrid(a=wp, options=grid_options)
    return wp

jp.justpy(grid_test)
```

**Comment by elimintz** [comment]
The newly released version 0.1.4 supports this:
https://justpy.io/grids_tutorial/field_evaluation/

Please let me know if it works for you

**Comment by giodegas** [comment]
great thank's, like a charm!
